### ISSUE TITLE
New Bourbon library wont compile and generates an error

### ISSUE THREAD

**Issue by kkline** [bdkjones/CodeKit-1-Old#704]
Codekit throws an error on the ../addons/_html5-input-types.scss file in the latest couple of Bourbon libraries saying that it cannot compile.  Here is the text from the error.  It seems to be having a problem with the !global declaration, as when I comment that out, it works fine.

Error:
Syntax error: Invalid CSS after "...-type), comma) ": expected "}", was "!global;"
        on line 25 of /Documents/Prototypes/test-proj/stylesheets/bourbon/addons/_html5-input-types.scss
        from line 59 of /Documents/Prototypes/test-proj/stylesheets/bourbon/_bourbon.scss

**Comment by bdkjones** [comment]
Have you switched CodeKit to an external Sass compiler?

**Comment by kkline** [comment]
Nope... I am using the built in Sass compiler.  I just downloaded a new version of the Bourbon library to take advantage of the latest flexbox spec mixins.

**Comment by bdkjones** [comment]
This looks like an error in Bourbon. Please check with the Bourbon team and try compiling the file using Sass 3.2.12 (the latest stable release) at the command line.
